treeherder/etl/common.py

```diff
@@ -10,6 +11,8 @@ def make_request(url, method='GET', headers=None, timeout=30, **kwargs):
     """A wrapper around requests to set defaults & call raise_for_status()."""
     headers = headers or {}
     headers['User-Agent'] = 'treeherder/{}'.format(settings.SITE_HOSTNAME)
+    if os.environ.get("GITHUB_TOKEN") and url.find("api.github.com") > -1:
```

Comment:
This guarantees that anywhere in the code that calls Github's API will add the correct headers assuming if `GITHUB_TOKEN` is defined.

Reply:
This removes the rate limiting of 60 calls an hour.